src/v2/Components/LotTimer.tsx

```diff
@@ -57,7 +63,11 @@ export const LotTimerFragmentContainer = createFragmentContainer(LotTimer, {
   `,
 })
 
-export const getTimerCopy = (time, hasStarted) => {
+export const getTimerCopy = (
+  time,
+  hasStarted,
+  extendedBiddingEndAt?: string | null
```

Comment:
I don't think this is an optional parameter, although the type looks correct (it can be `null` before bidding has been extended).

Reply:
I made it an optional parameter as there seems to be a few other places where we use this component and I wasn't sure we would want that extended label on it in those circumstances, but I can go ahead and update them as well.

Reply:
Ah, interesting, hadn't realized that! It looks like the `getTimerCopy` helper is used in one other location (sort of odd to reach into here and use a helper method like that!), so in that case 👍